src/connectors/radio7.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+Connector.playerSelector = '#page-wrap';
+
+Connector.artistTrackSelector = '#now';
+
+Connector.isPlaying = () => Util.isElementVisible('#jp_container_1 .jp-pause');
```

Comment:
It's better to use `Connector.pauseButtonSelector`.

Reply:
Crazy - I was trying with this yesterday and it wasn't working :D